### ISSUE TITLE
java.lang.IllegalArgumentException: Unsupported class file major version 57 - Guice 4.2.2

### ISSUE THREAD

**Issue by Razormx90** [playframework/playframework#10197]
### Play Version

2.8.1

### API

Java

### Operating System

Mac OS Catalina - 10.15.3

### JDK

java version "14.0.1" 2020-04-14
Java(TM) SE Runtime Environment (build 14.0.1+7)
Java HotSpot(TM) 64-Bit Server VM (build 14.0.1+7, mixed mode, sharing)

### Library Dependencies

guice 4.2.2

### Expected Behavior

1. Request a page that displays a list of items taken from a DB
2. Show the page

### Actual Behavior

The page renders a 500 error code with no body content

This is the error i get in sbt:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/Users/me/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/inject/guice/4.2.2/guice-4.2.2.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[info] p.a.d.DefaultDBApi - Database [default] initialized at jdbc:mysql://localhost:3306/db?verifyServerCertificate=false&useSSL=true
[info] p.a.d.HikariCPConnectionPool - Creating Pool for datasource 'default'
[error] a.a.ActorSystemImpl - Internal server error, sending 500 response
com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalArgumentException: Unsupported class file major version 57
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2051)
	at com.google.common.cache.LocalCache.get(LocalCache.java:3953)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3976)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4960)
	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4966)
	at com.google.inject.internal.util.StackTraceElements.forMember(StackTraceElements.java:71)
	at com.google.inject.internal.Messages.formatSource(Messages.java:241)
	at com.google.inject.internal.Messages.formatSource(Messages.java:220)
	at com.google.inject.internal.Messages.formatInjectionPoint(Messages.java:276)
	at com.google.inject.internal.Messages.formatSource(Messages.java:235)
Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 57
	at com.google.inject.internal.asm.$ClassReader.<init>(ClassReader.java:184)
	at com.google.inject.internal.asm.$ClassReader.<init>(ClassReader.java:166)
	at com.google.inject.internal.asm.$ClassReader.<init>(ClassReader.java:152)
	at com.google.inject.internal.asm.$ClassReader.<init>(ClassReader.java:273)
	at com.google.inject.internal.util.LineNumbers.<init>(LineNumbers.java:69)
	at com.google.inject.internal.util.StackTraceElements$1.load(StackTraceElements.java:49)
	at com.google.inject.internal.util.StackTraceElements$1.load(StackTraceElements.java:45)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155)
[error] p.a.h.DefaultHttpErrorHandler - 

! @7ffaoc05e - Internal server error, for (GET) [/robots.txt?1587030345723] ->
 
play.api.PlayException: Not initialized[?]
	at play.core.server.DevServerStart$$anon$1.<init>(DevServerStart.scala:124)
	at play.core.server.DevServerStart$.$anonfun$mainDev$1(DevServerStart.scala:117)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:22)
	at play.core.server.DevServerStart$.mainDev(DevServerStart.scala:76)
	at play.core.server.DevServerStart$.mainDevHttpMode(DevServerStart.scala:50)
	at play.core.server.DevServerStart.mainDevHttpMode(DevServerStart.scala)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)

[warn] Canceling execution...
```

**Comment by mkurz** [comment]
The problem here is your are using Java 14. However Play officially supports Java 8 and 11 only.
The specific problem here is that guice comes with an asm version that does not support Java 14 yet.
The fix is to use Java 11.

**Comment by Razormx90** [comment]
Retried 3 times changing Java each time These are the version I tried:

java version "1.8.0_45"
Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)

java version "1.8.0_251"
Java(TM) SE Runtime Environment (build 1.8.0_251-b08)
Java HotSpot(TM) 64-Bit Server VM (build 25.251-b08, mixed mode)

java version "11.0.7" 2020-04-14 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.7+8-LTS)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.7+8-LTS, mixed mode)

**The issue persist:**

I'm just trying to run the ebean example without any add/edit on files by myself

(```
Server started, use Enter to stop and go back to the console...)

java.lang.IllegalArgumentException: Unsupported class file major version 57
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:184)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:166)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:152)
	at io.ebean.enhance.Transformer.detect(Transformer.java:352)
	at io.ebean.enhance.Transformer.enhanceEntityAndTransactional(Transformer.java:187)
	at io.ebean.enhance.Transformer.transform(Transformer.java:152)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:57)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:41)
	at io.ebean.enhance.ant.OfflineFileTransform.transformFile(OfflineFileTransform.java:149)
	at io.ebean.enhance.ant.OfflineFileTransform.processPackage(OfflineFileTransform.java:133)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:102)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:71)
	at play.ebean.sbt.PlayEbean$.$anonfun$ebeanEnhance$1(PlayEbean.scala:84)
	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:290)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
java.lang.IllegalArgumentException: Unsupported class file major version 57
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:184)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:166)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:152)
	at io.ebean.enhance.Transformer.detect(Transformer.java:352)
	at io.ebean.enhance.Transformer.enhanceEntityAndTransactional(Transformer.java:187)
	at io.ebean.enhance.Transformer.transform(Transformer.java:152)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:57)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:41)
	at io.ebean.enhance.ant.OfflineFileTransform.transformFile(OfflineFileTransform.java:149)
	at io.ebean.enhance.ant.OfflineFileTransform.processPackage(OfflineFileTransform.java:133)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:102)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:71)
	at play.ebean.sbt.PlayEbean$.$anonfun$ebeanEnhance$1(PlayEbean.scala:84)
	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:290)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
java.lang.IllegalArgumentException: Unsupported class file major version 57
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:184)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:166)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:152)
	at io.ebean.enhance.Transformer.detect(Transformer.java:352)
	at io.ebean.enhance.Transformer.enhanceEntityAndTransactional(Transformer.java:187)
	at io.ebean.enhance.Transformer.transform(Transformer.java:152)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:57)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:41)
	at io.ebean.enhance.ant.OfflineFileTransform.transformFile(OfflineFileTransform.java:149)
	at io.ebean.enhance.ant.OfflineFileTransform.processPackage(OfflineFileTransform.java:133)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:102)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:71)
	at play.ebean.sbt.PlayEbean$.$anonfun$ebeanEnhance$1(PlayEbean.scala:84)
	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:290)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
java.lang.IllegalArgumentException: Unsupported class file major version 57
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:184)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:166)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:152)
	at io.ebean.enhance.Transformer.detect(Transformer.java:352)
	at io.ebean.enhance.Transformer.enhanceEntityAndTransactional(Transformer.java:187)
	at io.ebean.enhance.Transformer.transform(Transformer.java:152)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:57)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:41)
	at io.ebean.enhance.ant.OfflineFileTransform.transformFile(OfflineFileTransform.java:149)
	at io.ebean.enhance.ant.OfflineFileTransform.processPackage(OfflineFileTransform.java:133)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:102)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:71)
	at play.ebean.sbt.PlayEbean$.$anonfun$ebeanEnhance$1(PlayEbean.scala:84)
	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:290)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
java.lang.IllegalArgumentException: Unsupported class file major version 57
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:184)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:166)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:152)
	at io.ebean.enhance.Transformer.detect(Transformer.java:352)
	at io.ebean.enhance.Transformer.enhanceEntityAndTransactional(Transformer.java:187)
	at io.ebean.enhance.Transformer.transform(Transformer.java:152)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:57)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:41)
	at io.ebean.enhance.ant.OfflineFileTransform.transformFile(OfflineFileTransform.java:149)
	at io.ebean.enhance.ant.OfflineFileTransform.processPackage(OfflineFileTransform.java:133)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:102)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:71)
	at play.ebean.sbt.PlayEbean$.$anonfun$ebeanEnhance$1(PlayEbean.scala:84)
	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:290)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
java.lang.IllegalArgumentException: Unsupported class file major version 57
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:184)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:166)
	at io.ebean.enhance.asm.ClassReader.<init>(ClassReader.java:152)
	at io.ebean.enhance.Transformer.detect(Transformer.java:352)
	at io.ebean.enhance.Transformer.enhanceEntityAndTransactional(Transformer.java:187)
	at io.ebean.enhance.Transformer.transform(Transformer.java:152)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:57)
	at io.ebean.enhance.common.InputStreamTransform.transform(InputStreamTransform.java:41)
	at io.ebean.enhance.ant.OfflineFileTransform.transformFile(OfflineFileTransform.java:149)
	at io.ebean.enhance.ant.OfflineFileTransform.processPackage(OfflineFileTransform.java:133)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:102)
	at io.ebean.enhance.ant.OfflineFileTransform.process(OfflineFileTransform.java:71)
	at play.ebean.sbt.PlayEbean$.$anonfun$ebeanEnhance$1(PlayEbean.scala:84)
	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
	at sbt.Execute.work(Execute.scala:290)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[error] p.a.h.DefaultHttpErrorHandler - 

! @7ffbpjhlo - Internal server error, for (GET) [/] ->
 
play.api.UnexpectedException: Unexpected exception[UnsupportedClassVersionError: controllers/HomeController has been compiled by a more recent version of the Java Runtime (class file version 57.0), this version of the Java Runtime only recognizes class file versions up to 55.0]
	at play.core.server.DevServerStart$$anon$1.reload(DevServerStart.scala:215)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:142)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:300)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:190)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1289)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:624)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:501)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:599)
Caused by: java.lang.UnsupportedClassVersionError: controllers/HomeController has been compiled by a more recent version of the Java Runtime (class file version 57.0), this version of the Java Runtime only recognizes class file versions up to 55.0
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1016)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
	at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:550)
	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:458)
	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:452)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:451)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[error] p.a.h.DefaultHttpErrorHandler - 

! @7ffbpjhlo - Internal server error, for (GET) [/robots.txt?1587042807222] ->
 
play.api.UnexpectedException: Unexpected exception[UnsupportedClassVersionError: controllers/HomeController has been compiled by a more recent version of the Java Runtime (class file version 57.0), this version of the Java Runtime only recognizes class file versions up to 55.0]
	at play.core.server.DevServerStart$$anon$1.reload(DevServerStart.scala:215)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:142)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:300)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:190)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1289)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:624)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:501)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:599)
Caused by: java.lang.UnsupportedClassVersionError: controllers/HomeController has been compiled by a more recent version of the Java Runtime (class file version 57.0), this version of the Java Runtime only recognizes class file versions up to 55.0
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1016)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
	at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:550)
	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:458)
	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:452)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:451)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
```

**Comment by mkurz** [comment]
Remove the `target` folder before trying again.

**Comment by mkurz** [comment]
This will be fixed by #10724 - Guice 5 supports Java class files for Java 17 already!
